charts/sn-platform/templates/control-center/_control_center.tpl

```diff
@@ -93,4 +102,13 @@ pulsar controller ingress target port for http endpoint
     {{- else -}}
         {{- print "false" -}}
     {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+
+{{- define "pulsar.ingress.image" -}}
+    {{- if and (eq (include "pulsar.kubeVersion.isLessThanV122" .) "false") (semverCompare "< 1.0.0" .Values.images.nginx_ingress_controller.tag )}}
+        {{- print "k8s.gcr.io/ingress-nginx/controller:v1.1.1"}}
```

Comment:
It's a bit hacky, could you add some comments here?

Reply:
Done